models/presentation.go

```diff
@@ -0,0 +1,32 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Presentation represents a presentation/session
+type Presentation struct {
+	ID               int       `json:"id" db:"id"`
+	CreatedAt        time.Time `json:"created_at" db:"created_at"`
+	UpdatedAt        time.Time `json:"updated_at" db:"updated_at"`
+	Title            string    `json:"title" db:"title"`
+	ShortDescription string    `json:"short_description" db:"short_description"`
+	LongDescription  string    `json:"long_description" db:"long_description"`
+	SpeakerID        int       `json:"speaker_id" db:"speaker_id"`
```

Comment:
If we have the join table, do we even need SpeakerID in the presentation table at all?